src/diff/index.js

```diff
@@ -288,15 +288,15 @@ export function applyRef(ref, value, ancestorComponent) {
  * @param {import('../internal').Component} ancestorComponent The parent
  * component to this virtual node
  */
-export function unmount(vnode, ancestorComponent) {
+export function unmount(vnode, ancestorComponent, skipRemove) {
 	let r;
```

Comment:
In this function the `r` variable is used to refer to 5 different things (vnode ref, mounted DOM node, Component instance, last rendered output of component, array of child vnodes). Is this overloading for code golfing reasons?

Reply:
yup, overloading the variable saves a few bytes.